decode/loadjpeg.c

```diff
@@ -74,7 +75,9 @@ int convert_one_image(const char *infilename)
   buf = (unsigned char *)malloc(length_of_file + 4);
   if (buf == NULL)
     exitmessage("Not enough memory for loading file\n");
-  fread(buf, length_of_file, 1, fp);
+  n_items = fread(buf, length_of_file, 1, fp);
+  if (n_items != 1)
```

Comment:
if `n_items` is only used once, then why not just do:
```
if (1 ! = fread(buf, length_of_file, 1, fp))
```
... instead of creating an extra variable.